### PR TITLE
rename loopy

### PR DESCRIPTION
**Pull request by matthiasdiener**
cc https://github.com/illinois-ceesd/mirgecom/pull/136

### CODE REVIEW

**Comment by inducer** [comment]
I saw this too, but:

- It'll keep working in the presence of the rename.
- Removing it now will break the workflow of folks who run a new emirge with an old mirgecom.

So I think removing this is premature.

**Comment by matthiasdiener** [comment]
> I saw this too, but:
> 
> * It'll keep working in the presence of the rename.
> * Removing it now will break the workflow of folks who run a new emirge with an old mirgecom.
> 
> So I think removing this is premature.

Ok, but I'll document this at least.

**Comment by inducer** [comment]
> Ok, but I'll document this at least.

Fair point.

**Comment by matthiasdiener** [comment]
ready for review.

File: version.sh [hunk 1]
```diff
@@ -58,6 +58,8 @@ for i in "${!module_names[@]}"; do
         branchname_git="($(git rev-parse --abbrev-ref HEAD))"
         cd ..
     elif [[ $name == "loopy" && -d loo-py ]]; then
+        # FIXME: this is a hack that was needed before loo-py was renamed to loopy;
+        # remove this by 12/2020.
         cd loo-py || exit 1
```
Comment (inducer) [inline version.sh:63]:
```suggestion
        # remove this by 12/2021.
        # https://github.com/illinois-ceesd/emirge/pull/85
        cd loo-py || exit 1
```

**Review by inducer — APPROVED** [review]

**Review by inducer — APPROVED** [review]

**Pull request merged**